code/src/components/Images.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+// Component for the image and icons on top
+
+export const Images = props => {
```

Comment:
It is a convention to name your component in plural form when if it contains a collection of other components. 
`Posts` contains a lot of  `Post`. So here it should be `Image`